docs/pywbemcli/mock_support.rst

```diff
@@ -90,10 +89,10 @@ The following types of files are supported for the ``--mock-server`` option:
       A flag that contains the value of the boolean
       :ref:`--verbose general option` of pywbemcli.
 
-  * Old-style: The mock script does not have a ``setup()`` function. This
-    approach is not recommended, but it is supported on all supported Python
-    versions. Using old-style mock scripts in a connection definition prevents
-    caching of its mock environment.
+  * Old-style(Python < 3.5): The mock script does not have a ``setup()`` function.
```

Comment:
The old style is supported on all Python versions. Only the new style requires Python>=3.5.

Reply:
whoops, got a bit carried away.  Fixed DONE